crypto/fipsmodule/evp/evp.c

```diff
@@ -156,6 +158,68 @@ int EVP_PKEY_cmp(const EVP_PKEY *a, const EVP_PKEY *b) {
   return -2;
 }
 
+char *EVP_get_pw_prompt(void) {
+    return (char*)"Enter PEM passphrase:";
+}
+
+int EVP_read_pw_string(char *buf, int length, const char *prompt, int verify) {
+  return EVP_read_pw_string_min(buf, 0, length, prompt, verify);
+}
+
+int EVP_read_pw_string_min(char *buf, int min_length, int length,
+                           const char *prompt, int verify) {
+  int ret = -1;
+  char verify_buf[1024];
```

Comment:
* Proactively zero the caller's `buf`?
* Also `OPENSSL_cleanse(verify_buf, sizeof(verify_buf`)
* Fail if `min_length >= 1024`
* Fail if `min_length >= length`